scripts/setup_for_wrf.py

```diff
@@ -16,102 +16,57 @@
 import stat
 import netCDF4
 from setup_runs.wrf.fetch_fnl import download_gdas_fnl_data
+from setup_runs.config_read_functions import (read_config_file,
+                                              parse_config,
+                                              add_environment_variables,
+                                              substitute_variables,
+                                              parse_boolean_keys,
+                                              process_date_string,
+                                              load_wrf_config)
 
 ## get command line arguments
 parser = argparse.ArgumentParser()
 
-parser.add_argument("-c", "--configFile", help="Path to configuration file", default = 'config.nci.json')
+# TODO: change default file path back to `config.nci.json`
+parser.add_argument("-c", "--configFile", help="Path to configuration file", default = '../config.nci.json')
 args = parser.parse_args()
 configFile = args.configFile
 
-## read the config file
-assert os.path.exists(configFile), "No configuration file was found at {}".format(configFile)
+wrf_config = load_wrf_config(configFile)
 
-try:
-    f = open(configFile,'rt')
-    input_str = f.read()
-    f.close()
-except Exception as e:
-    print("Problem reading in configuration file")
-    print(str(e))
-    sys.exit()
+config = wrf_config.__getstate__()
```

Comment:
@lewisjared do we want to change the object back to a dict? Or would it be better to call the individual values of the config directly like: 'wrf_config.start_date'

Reply:
Yes for now. Eventually we can fix the config everywhere, but that is a good change to do as a single PR.

```suggestion
config = attrs.asdict(wrf_config)
```